experimental/ModelBuilder/test/TestMatMulVulkan.cpp

```diff
@@ -114,8 +114,12 @@ void testMatMul() {
     spirvModulePM.addPass(
         mlir::spirv::createUpdateVersionCapabilityExtensionPass());
 
-    pm.addPass(
-        mlir::createAddVulkanLaunchWrapperPass({width, height, 1}, args));
+    int numWorkgroupX =
+        vWorkgroupSizes.empty() ? 1 : width / vWorkgroupSizes[0];
```

Comment:
Don't we need to accommodate for the irregular cases by adding one workgroup? Are we assuming always 2^N here?

Reply:
True, I think it works without it since we insert a loop anyway but I changed it as it is better this way.